scripts/rollup/results.json

```diff
@@ -25,28 +25,28 @@
       "gzip": 6703
     },
     "react-dom.development.js (UMD_DEV)": {
-      "size": 622587,
-      "gzip": 143511
+      "size": 631901,
+      "gzip": 144934
     },
     "react-dom.production.min.js (UMD_PROD)": {
-      "size": 100874,
-      "gzip": 31801
+      "size": 101064,
+      "gzip": 31915
     },
     "react-dom.development.js (NODE_DEV)": {
-      "size": 584856,
-      "gzip": 134799
+      "size": 594156,
+      "gzip": 136217
     },
     "react-dom.production.min.js (NODE_PROD)": {
-      "size": 105280,
-      "gzip": 33069
+      "size": 107348,
```

Comment:
@gaearon Interestingly this still adds a bit. I wonder what it is.

Reply:
Probably invariant messages that will get stripped when we extract errors.
You can try 

```
yarn build --extract-errors
yarn build
```

(yea, twice)

Reply:
Why twice?

I actually only run this step once in the release script, after testing the output of `yarn build` vs `yarn build --extract-errors` and determining they were the same.

Did I overlook something?

Reply:
I thought the first time it updates the error code file, and second time it rebuilds taking it into account. Maybe that's how it used to work in the past.

Reply:
Cool. Thanks for elaborating.

I'll test again to be sure.

Reply:
Okay. Just did a quick test locally with a revision that had a stale error codes JSON file.

Comparing the output of `yarn build -- --error-codes` (run once) to `yarn build` (run again after the error codes were updated) does seem to yield slightly different results. eg for `packages/react-dom/cjs/react-dom.development.js`:
```diff
12469c12469
<                 invariant(isBatchingUpdates, 'Task updates can only be scheduled as a nested update or ' + 'inside batchedUpdates. This error is likely caused by a ' + 'bug in React. Please file an issue.');
---
>                 !isBatchingUpdates ? invariant(false, 'Task updates can only be scheduled as a nested update or inside batchedUpdates. This error is likely caused by a bug in React. Please file an issue.') : void 0;
```

These 2 look equivalent to me.

I also did a more extreme case where I completely emptied out the error codes JSON and ran a single `yarn build -- --error-codes` step and then compared output. The only changes were ones like I pasted above.

I think this means it's okay to combine these steps?

Reply:
I guess the second format is slightly better because it potentially avoids creating strings (eg `getComponentName`) that we wouldn't actually use if the invariant passes (which is the common case) but I'm confused as to what the relationship is between the error-codes JSON and this output.

Reply:
I don't remember :-)

Reply:
This is because of our build script's `replace-invariant-error-code` short-circuits if it can't find the error in the error codes map. I can fix that to still do the replacement.

Reply:
Would be great.

Reply:
Yup. I'm on it 👍